simd/src/main/java/com/dylibso/chicory/simd/SimdInterpreterMachine.java

```diff
@@ -15,10 +15,11 @@
 public class SimdInterpreterMachine extends InterpreterMachine {
```

Comment:
Can you try making this `final`? I doubt it will have any impact on performance, but it doesn't hurt, and we'd expect the class to be final anyway.